assets/queries/k8s/default_service_account_binding/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Default_Service_Account_Binding",
+  "queryName": "Kubernetes Service Account Binding is Default",
```

Comment:
I think "Kubernetes" is not necessary. A better name could be "Role Binding To Default Service Account"